### PR TITLE
Android build

### PR DESCRIPTION
**Pull request by JW-CH**

### CODE REVIEW

**Comment by BlakeFL** [comment]
If you need an Android beta tester, let me know.  I have an old Fire Tablet waiting for an APK.

**Comment by JW-CH** [comment]
> If you need an Android beta tester, let me know. I have an old Fire Tablet waiting for an APK.

Thank you. Feel free to create issues once we are happy with the state of the android build ;)

File: ImmichFrame/Models/Settings.cs [hunk 1]
```diff
@@ -43,89 +43,65 @@ public static Settings CurrentSettings
     {
         get
         {
-            if (PlatformDetector.IsAndroid())
+            if (_settings == null)
             {
-                if (_settings == null)
+                if (File.Exists(AppDomain.CurrentDomain.BaseDirectory + @"Settings.xml"))
```
Comment (JW-CH) [inline ImmichFrame/Models/Settings.cs:48]: Have something like Settings.IsFromFile? This is nicer for maintainability

**Comment by JW-CH** [comment]
Hey @BlakeFL, if you want to, there is a prerelease version for android: https://github.com/3rob3/ImmichFrame/releases/tag/v1.0.0.10-alpha apk is attached to the release.

**Comment by BlakeFL** [comment]
> Hey @BlakeFL, if you want to, there is a prerelease version for android: https://github.com/3rob3/ImmichFrame/releases/tag/v1.0.0.10-alpha apk is attached to the release.

I will try it out later today.  Thanks!

**Comment by BlakeFL** [comment]
The apk worked well.  It is showing images from my server and the weather api is working.  A few observations though...

-  I messed up the settings the first attempt and there was no prompt to return back to that screen.  I closed out of the app a few times and was eventually taken back and I corrected the issue.  
- On the settings page all the text boxes are very small for me.
- For the weather, I don't believe the city name is necessary.  To reduce vertical space, an image might be better next to the temperature instead of "Clear Sky" below.  I also do not care about decimals in the temp so that could free up space as well.

**Comment by 3rob3** [comment]
- You can get back to setting at any time by pressing/clicking the upper middle quadrant. The documentation will be updated once we merge this.
- What size device and resolution? This is going to be tough to get perfect on all devices unfortunately.
- I mostly agree with the city comment, might need to make it an optional setting. Not sure about decimal places in temperature, it might be useful when showing Celsius. I personally like the text description for current conditions over a tiny icon.

**Comment by JW-CH** [comment]
image
Updated the textboxes, I wanted to do this before :)

For the other things: these are not really android related. I'd prefer to leave them right now and create new issues for them.

**Pull request merged**